src/main/interface/settings.c

```diff
@@ -686,6 +686,7 @@ const clivalue_t valueTable[] = {
     { "gps_sbas_mode",              VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_GPS_SBAS_MODE }, PG_GPS_CONFIG, offsetof(gpsConfig_t, sbasMode) },
     { "gps_auto_config",            VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_GPS_CONFIG, offsetof(gpsConfig_t, autoConfig) },
     { "gps_auto_baud",              VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_GPS_CONFIG, offsetof(gpsConfig_t, autoBaud) },
+    { "gps_use_galileo",            VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_GPS_CONFIG, offsetof(gpsConfig_t, useGalileo) },
```

Comment:
Can we make this `gps_ublox_use_galileo`, to make it obvious that it only supports ublox?

Also, coding standard is for the field that the parameter is stored in to use snake_case and an identical name as the name exposed in CLI, i.e. `gps_ublox_use_galileo`.

Reply:
I didn't add ublox word because maybe someone in the future can add something similar for NMEA or other. 
The no snake was to be aligned with others gps params.

Reply:
We should name it `gps_ublox_use_galileo` for now then. We can always rename it when it is implemented for NMEA. Otherwise we'll get issues reported like 'why does this not work on my NMEA GPS'.